Lang/French.pj.Lang

```diff
@@ -316,59 +316,59 @@
 
 //Save Type
 #620#  "Par défaut"
-#621#  "4-kbit EEPROM"
-#622#  "16-kbit EEPROM"
-#623#  "32-kbyte SRAM"
+#621#  "4 kbit EEPROM"
+#622#  "16 kbit EEPROM"
+#623#  "32 ko SRAM"

```

Comment:
He's right : "de" doesn't mean anything in this situation because it's a technical specification.
But I think, we should capitalize the "k" in "Ko".